apps/src/applab/Visualization.jsx

```diff
@@ -20,6 +20,16 @@ var styles = {
   },
   phoneFrame: {
     marginBottom: 0
+  },
+  screenBlock: {
+    backgroundColor: 'rgba(0, 0, 0, 0.05)',
```

Comment:
This one is animated with a nice occasional desync :) http://codepen.io/meduzen/pen/zxbwRV